components/style/properties/shorthand/mask.mako.rs

```diff
@@ -120,6 +120,30 @@
          })
     }
 
+    fn mask_position_to_css<W>(x: &HorizontalPosition,
+                               y: &VerticalPosition,
+                               dest: &mut W)
+                               -> fmt::Result where W: fmt::Write {
```

Comment:
Did you mean to create a public function to handle this in `values::specified::position` and use it in background and mask?